sympy/polys/matrices/tests/test_ddm.py

```diff
@@ -365,3 +365,36 @@ def test_DDM_charpoly():
 
     A = DDM([[ZZ(1), ZZ(2)]], (1, 2), ZZ)
     raises(DDMShapeError, lambda: A.charpoly())
+
+
+def test_DDM_getitem():
+    dm = DDM([
+        [ZZ(1), ZZ(2), ZZ(3)],
+        [ZZ(4), ZZ(5), ZZ(6)],
+        [ZZ(7), ZZ(8), ZZ(9)]], (3, 3), ZZ)
+
+    assert dm[1][1] == ZZ(5)
+    assert dm[1][-2] == ZZ(5)
+    assert dm[-1][-3] == ZZ(7)
```

Comment:
Should these not be testing `getitem` rather than `__getitem__`?

Reply:
Sorry, my bad.
Thanks.